reactions/examples/validate.cpp

```diff
@@ -49,18 +50,20 @@ string line;
             mytrans.push_back(lineData);         // add row to allData
     }
 
+
 // Populate arrays and normalise to 1 at large scales for Cosmology class input
 int Nkr = mytrans.size();
 int* Nkt = &Nkr;
 array Tm(*Nkt);
 array Tcb(*Nkt);
 array Tnu(*Nkt);
 array ki(*Nkt);
+
           for(int i = 0; i< Nkr; i++){
                   ki[i] = mytrans[i][0];
-                  Tm[i] = mytrans[i][6]/mytrans[0][6];
-                  Tcb[i] = mytrans[i][7]/mytrans[0][7];
-                  Tnu[i] = mytrans[i][5]/mytrans[0][5]; // should be adjusted if this column is all 0s ....
+                  Tm[i] = mytrans[i][6];
+                  Tcb[i] = mytrans[i][7];
+                  Tnu[i] = mytrans[i][5]; // should be adjusted if this column is all 0s ....
```

Comment:
I don't understand what this comment means exactly?

Reply:
I was getting some run errors when there was no neeutrino powerr in the transfer functtion. This seems to have gone away for some reason though. Can remove comment.